src/Context/GlobalContext/StateContext.jsx

```diff
@@ -1,72 +1,83 @@
-import { createContext, useContext, useReducer, useEffect, useState } from "react"
-import axios from "axios"
-import { uniqueCategory } from "../../utils/uniqueCategory"
-import { sortByPrice } from "../../utils/sortByPrice"
-import { stateReducerFun } from "../GlobalContext/StateReducer"
-
-const StateContext = createContext()
-
+import {
+    createContext,
+    useContext,
+    useReducer,
+    useEffect,
+    useState,
+} from "react";
+import axios from "axios";
+import { uniqueCategory } from "../../utils/uniqueCategory";
```

Comment:
We can use barrel export here. Export all methods from index.js inside the utils folder.